fs/mount/fs_procfs_mount.c

```diff
@@ -268,6 +268,26 @@ static int blocks_entry(FAR const char *mountpoint,
                 statbuf->f_blocks - statbuf->f_bavail, statbuf->f_bavail,
                 mountpoint);
 #else
+#  if defined(CONFIG_FS_LARGEFILE) && defined(CONFIG_HAVE_LONG_LONG)
```

Comment:
@ALTracer I'm confuse here, shouldn't be if defined(CONFIG_FS_LARGEFILE) && defined(CONFIG_LIBC_LONG_LONG)

Reply:
Well no, in this branch I'm trying to gracefully cast down and print as ul, but still check first for possible overflow since I can compute in ull. 
This would be impossible if HAVE_LONG_LONG got disabled.